code/backend/batch/utilities/helpers/ConfigHelper.py

```diff
@@ -125,16 +158,103 @@ def get_default_config():
 
 Question: {question}
 Answer:""",
```

Comment:
This PR removes the prompt to respond to the user in the user's language, the previous prompt had this line: "When directly replying to the user, always reply in the language the user is speaking."

Can we please add this line to the end of the new prompt? That way we can get the Wednesday team to review it together with the rest of the prompt. Thanks!

Reply:
Added

Reply:
Beautiful, thank you!